Applications/PageRank.cpp

```diff
@@ -162,6 +165,10 @@ int main(int argc, char **argv)
         double pr_time = timer.elapsed(false);
         MAIN_COUT("PageRank stopped after " << iter << " iterations in " << pr_time << " seconds" << std::endl);
 
+        std::string timing_output = input_graph;
+        timing_output.replace(timing_output.find_last_of('.'), std::string::npos, ".pr_time.csv");
```

Comment:
TODO: this is pretty hacky and might be annoying to deal with later - should factor out and make more robust